data/systemd/Makefile.am

```diff
@@ -19,6 +19,7 @@ systemddir = $(prefix)/lib/systemd/system
 generatordir = $(prefix)/lib/systemd/system-generators
 
 dist_systemd_DATA = anaconda.service \
+					anaconda-boss.service \
```

Comment:
Please fix this indentation.

Reply:
Fixed (looks like tabs instead of spaces get there somehow - weird).